### PR TITLE
(sourcetree-disableautoupdate) Add package to disable auto-update of sourcetree

### PR DESCRIPTION
**Pull request by pascalberger**
Add package for disabling auto-update functionality in SourceTree

### CODE REVIEW

File: manual/sourcetree-disableautoupdate/sourcetree-disableautoupdate.nuspec [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <projectSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/manual/sourcetree-disableautoupdate</projectSourceUrl>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/manual/sourcetree-disableautoupdate</packageSourceUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/d9984ffd9b6481c92e83fc1ec16f55184633adb6/icons/sourcetree.png</iconUrl>
+    <licenseUrl>https://opensource.org/licenses/MIT</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>SourceTree Auto Update Deactivation</summary>
+    <description>Disables the auto update functionality of SourceTree.</description>
+    <tags>sourcetree cross-platform</tags>
+    <dependencies>
+      <dependency id="sourcetree"/>
```
Comment (AdmiringWorm) [inline manual/sourcetree-disableautoupdate/sourcetree-disableautoupdate.nuspec]: as per [1.1.11 Dependency versions](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#1111-dependency-versions) you should specify a minimum version.

File: manual/sourcetree-disableautoupdate/sourcetree-disableautoupdate.nuspec [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <authors>chocolatey</authors>
+    <owners>chocolatey, Pascal Berger</owners>
+    <projectSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/manual/sourcetree-disableautoupdate</projectSourceUrl>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/manual/sourcetree-disableautoupdate</packageSourceUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/d9984ffd9b6481c92e83fc1ec16f55184633adb6/icons/sourcetree.png</iconUrl>
+    <licenseUrl>https://opensource.org/licenses/MIT</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>SourceTree Auto Update Deactivation</summary>
+    <description>Disables the auto update functionality of SourceTree.</description>
+    <tags>sourcetree cross-platform</tags>
```
Comment (AdmiringWorm) [inline manual/sourcetree-disableautoupdate/sourcetree-disableautoupdate.nuspec]: I wonder if perhaps adding a config (or configuration) tag here would be helpfull.
Thoughts?

File: manual/sourcetree-disableautoupdate/tools/chocolateyInstall.ps1 [hunk 3]
```diff
@@ -0,0 +1,4 @@
+﻿$ErrorActionPreference = 'Stop'
+
+# Disable auto update
+New-Item 'HKCU:\Software\Caphyon\Advanced Updater\{138B87CA-4905-43FE-BB63-184FFB0CE1EC}\Settings' -Force | New-ItemProperty -Name AutoUpdatePolicy -Value 0 -Force
```
Comment (AdmiringWorm) [inline manual/sourcetree-disableautoupdate/tools/chocolateyInstall.ps1]: as per [2.2 Formatting](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#22-formatting), you're missing an empty line at the end of the file.

File: manual/sourcetree-disableautoupdate/tools/chocolateyUninstall.ps1 [hunk 4]
```diff
@@ -0,0 +1,4 @@
+﻿$ErrorActionPreference = 'Stop'
+
+# Enable auto update
+Remove-ItemProperty -Name 'AutoUpdatePolicy' -Path 'HKCU:\Software\Caphyon\Advanced Updater\{138B87CA-4905-43FE-BB63-184FFB0CE1EC}\Settings'
```
Comment (AdmiringWorm) [inline manual/sourcetree-disableautoupdate/tools/chocolateyUninstall.ps1]: as per [2.2 Formatting](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#22-formatting), you're missing an empty line at the end of the file.

File: manual/sourcetree-disableautoupdate/sourcetree-disableautoupdate.nuspec (hunk 2 above)
Comment (pascalberger) [inline manual/sourcetree-disableautoupdate/sourcetree-disableautoupdate.nuspec]: @AdmiringWorm Also thought about this. Might make sense IMHO

**Pull request merged**